flamedisx/source.py

```diff
@@ -484,15 +557,14 @@ def mu_before_efficiencies(self, **params):
     # Functions you probably should override
     ##
 
-    def _annotate(self, _skip_bounds_computation=False):
+    def _annotate(self):
         """Add columns needed in inference to self.data
-        :param _skip_bounds_computation: Do not compute min/max bounds
-        TODO: explain why useful, see simulator
         """
         pass
 
     def add_extra_columns(self, data):
         """Add additional columns to data
+        If the final signals are required to be present, use
```

Comment:
use what

Reply:
Good question? I don't remember; removed the line.